src/patients/GeneralInformation.tsx

```diff
@@ -273,27 +246,27 @@ const NewPatientForm = (props: Props) => {
               name="address"
               value={patient.address}
               isEditable={isEditable}
-              onChange={(event: React.ChangeEvent<HTMLTextAreaElement>) => {
-                onFieldChange('address', event.currentTarget.value)
-              }}
+              onChange={
+                (event: React.ChangeEvent<HTMLTextAreaElement>) =>
+                  onFieldChange && onFieldChange('address', event.currentTarget.value)
+                // eslint-disable-next-line react/jsx-curly-newline
+              }
```

Comment:
Can't get eslint to accept this any way I can think to format it, or via the eslint automatic fix. There must be two conflicting rules.